kata-deploy/README.md

```diff
@@ -80,22 +80,28 @@ Workloads which utilize Kata can node-select based on ```katacontainers.io/kata-
 run through an applicable runtime if they are marked with the appropriate runtimeClass annotation.
 
 
-The following YAML snippet shows how to specify a workload should use Kata with QEMU:
-```
-spec:
-  template:
-    spec:
-      runtimeClassName: kata-qemu
-```
-
-The following YAML snippet shows how to specify a workload should use Kata with Firecracker:
-```
-spec:
-  template:
-    spec:
-      runtimeClassName: kata-fc
-```
-
+In order to use a workload Kata with QEMU, add a RuntimeClass as:
+- For k8s 1.14:
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/k8s-1.14/kata-qemu-runtimeClass.yaml
```

Comment:
I'm not convinced this is a good idea as although it might work today, what happens when 1.15 comes out? How many versions of k8s are you planning to continually update this doc and packaging scripts for? Also, this doesn't conform to https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.

Reply:
@jodh-intel This is something that needs to be updated as this feature moves from alpha to stable in kubernetes, causing a change in the version in the yaml files. 
We need to make sure we support at least the last two version of k8s. Overall we should be maintaining our yaml files for different versions of k8s and not just the latest.

Reply:
I suspect we'll end up with most 3 versions for the alpha/beta/full-release API version number/name changes, so I don't see this as a blocker. We may ultimately name them as `v1.14+` etc. to cover which of alpha/beta/etc. covers which version range.